notes/transcript10.md

```diff
@@ -0,0 +1,242 @@
+
+# Core Improvements
+12:00 - 13:00
+
+Oskar: You can take improvements as well while you're at it.
+
+Status: For the improvements there's a different situation there, there's no research there we are just implementing things. We implemented a quicker login so it's very very fast on at least my phone and even on the small phones it's very quick and it should be released in an internal release that we will have this week hopefully. We also added added a second password for Android so we have this feature as well as Iris but there is more things to do there so I don't consider it implemented completely because we also need to improve how it stores passwords in both places, so it's about 30% in.
+
+13:00 - 14:00
+
+Status: And also about half done with reliable messages so pruning was I guess already in status go but it needs to be deployed to work properly and there are few things like email confirmations and mail server management in general that needs to be done. We also started our dependency audits so we can build older versions of the application easier than right now but we just use some master or some current version of toolset. The last one is that there is some more work that was done with BFS and device sync and I think we will actually work on it like this week probably because it also might clash with tribute to talk so they need some coordination there, so that's all for us.
+
+# Protocal Update
+14:00 - 15:00
+
+Oskar: Cool thank you. I'll do the protocol update. So mainly two things have been done. So one is workshop preparation. We're gonna meet just before Fossum in Brussels end of January, we'll be fine to seven people from sellers. So working on agenda and making as useful to as many people as possible. The other thing is proof of concept for a sort of data sync layer. I've written a post about this before but the idea is that you encapsulate an in between layer that doesn't concern itself with a message per say but instead of just syncing a piece of data and that allows us to build things on top of it. Sort of public chat and group chat and so on, but below it you can swap whisper out and still be able to sync data in a peer-to-peer way. In this proof concept there is a proto-buff specification that is wrong in some places but gets the jist across.
+
+# Dao
+15:00 - 16:00
+
+Oskar: As well as a sort of basic simulation in Python that has some unreliability assumptions and also deals with mobile node having slightly different characteristics than having something like a desktop note where a mobile node is mostly offline but sometimes goes online. Then sort of for a one-to-one chat and seeing how the system behaves, so I did a short write-up, I'll link to it in the notes, yeah please ask questions or specific things you want me to try to answer with this concept. Please ping me as well. This week we will continue to work from the workshop and looking at what original direction the future will constantly bring. dao?
```

Comment:
"looking at what original direction the future will constantly bring" huh did I really say that :D doesn't make much sense